src/Output/Configurator/QueryConfigurator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tomaj\NetteApi\Output\Configurator;
+
+use Nette\Application\Request;
+
+class QueryConfigurator implements ConfiguratorInterface
+{
+    private $noSchemaValidateParam = "no_schema_validate";
+    private $errorDetailParam = "error_detail";
+
+    /**
+     * @param string $noSchemaValidateParam Name of get parameter to disable schema validation
+     * @param string $errorDetailParam Name of get parameter to show additional info in error response
+     */
+    public function __construct(string $noSchemaValidateParam = "no_schema_validate", string $errorDetailParam = "error_detail")
+    {
+        $this->noSchemaValidateParam = $noSchemaValidateParam;
+        $this->errorDetailParam = $errorDetailParam;
+    }
+
+    public function validateSchema(?Request $request = null): bool
+    {
+        if ($request === null) {
+            return false;
+        }
+        $getParams = $request->getParameters();
+        return !isset($getParams[$this->noSchemaValidateParam]);
```

Comment:
takze ?no_schema_validate=1 bude to iste ako ? no_schema_validate=0 hej? je to tak chcene?

Reply:
Skor som to chcel tak ze ?no_schema_validate&.... tiez vypne. Ale hej tympadom aj 0.

Reply:
ok sak ked si to tak chcel tak bude :)

Reply:
az teraz som si uvedomil ze to chceme asi naopak skor - defaultne vypnute a s debugom zapnut - rovnako ako je to pri tom DebuggerConfiguratore